.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I think you shouldn't add .vscode folder to your Pull Request or you can add .vscode to .gitignore